packages/flutter/lib/src/material/material_localizations.dart

```diff
@@ -32,6 +35,21 @@ class MaterialLocalizations {
   /// The tooltip for the [MonthPicker]'s "previous month" button.
   String get previousMonthTooltip => 'Previous month';
 
+  /// Provides geometric text preferences for the current locale.
+  ///
+  /// This text theme is incomplete. For example, it lacks text color
+  /// information. This theme must be merged with another text theme that
+  /// provides the missing values. The text styles provided by this theme are
+  /// inherited.
+  ///
+  /// Typically a complete theme is obtained via [Theme.of], which can be
+  /// localized using the [Localizations] widget.
+  ///
+  /// See also: https://material.io/guidelines/style/typography.html
+  TextTheme get localTextGeometry {
+    return _kLanguageTextGeometries[locale.languageCode] ?? _MaterialTextGeometry.englishLike;
```

Comment:
this should just be hard-coded to the englishLike text theme, since this is the en-US default localisation.